SourceryTests/Stub/JavaScriptTemplates/Includes.js

```diff
@@ -0,0 +1 @@
+<%- include('Equality.js') %>-%><%- include('Other.js') %>-%>
```

Comment:
When comparing the result of a template with just `<%- include('Equality.js') %>` to the result of the `Equality.js` itself I noticed that EJS adds an additional newline. To match the result files that were already existing without changing them, I added this. So without it the tests would fail 🙂